ramls/metadata-provider.raml

```diff
@@ -85,3 +87,30 @@ resourceTypes:
           body:
             text/plain:
               example: "Not found"
+  /jobLogEntries/{jobExecutionId}:
+    get:
+      is: [pageable]
+      description: get journal records by job execution id
+      queryParameters:
+        sortBy:
+          description: "sorting by field: source_record_order, title, source_record_action_status, instance_action_status, holdings_action_status, item_action_status, order_action_status, invoice_action_status, error"
+          example: source_record_order
+          type: string
+          required: false
+          default: source_record_order
+        order:
+          description: "sorting direction"
+          example: desc
+          enum: [asc, desc]
```

Comment:
is that the only place when we validate these param values?

Reply:
yes

Reply:
found it, but thanks for the comprehensive reply)

Reply:
@RuslanLavrov @KaterynaSenchenko But there is additional validation for "sortBy" parameter in JournalRecordDaoImpl:
`    if (sortBy != null && !jobLogEntrySortableFields.contains(sortBy)) {`

So, if it's needed, @RuslanLavrov could you please add validation for "order" also. Thank you!

Reply:
@VRohach, since `sortBy` has the default value and can not be null, sortBy != null validation rather is redundant, hence I'll remove it, thanks